packaging/ubuntu-14.04/snapd.service

```diff
@@ -6,6 +6,7 @@ Requires=snapd.socket
 ExecStart=/usr/lib/snapd/snapd
 EnvironmentFile=/etc/environment
 Restart=always
+Type=notify
```

Comment:
Intentional but I agree, it should be a separate PR. I will remove the commit 231035f92d79969c143f81acbd0df00bd6289b76 from this PR and propose separately.